packages/webview_flutter/webview_flutter_platform_interface/lib/src/platform_interface/webview_platform_callbacks_handler.dart

```diff
@@ -27,6 +27,8 @@ abstract class WebViewPlatformCallbacksHandler {
   /// /// Only works when [WebSettings.hasProgressTracking] is set to `true`.
   void onProgress(int progress);
 
+  Future<List<String>> onShowFileChooser();
```

Comment:
This is a breaking change as things are currently structured, so adding this will need to wait for the platform interface API update.

@mvanbeusekom @bparrishMines Can we verify that this kind of change will indeed by safe in the updated version?

Reply:
Yes, I don't see any reason this won't be possible with the new interface. The only difference is that we may make this unique to `AndroidWebViewControllerDelegate`. Which would only slightly change how it is used in this PR.

Reply:
@bparrishMines what's the branch of the "new interface" ? 
I understand that you are refactoring webview_flutter and you want to be sure that this PR will fit the new design.

Reply:
The work on the new interface can be tracked at: https://github.com/flutter/flutter/issues/94051.

Reply:
Ok, in the meantime, this PR is stale right ?